bagisto/bagisto
# Order id is having a lot of space Issue #5316 Fixed

## Issue Reference
#5316
## Description
Updated the CSS File for adjusting the width of Table 

## How To Test This?
1- Login with customer id.
2- Order a product and then check the order details on the customer's dashboard.
3- You will see, Now un-necessary space of order id is removed
